portal-bridge/src/types/mode.rs

```diff
@@ -82,6 +86,28 @@ impl FromStr for ModeType {
                     .map_err(|_| "Invalid bridge mode arg: block number")?;
                 Ok(ModeType::Block(block))
             }
+            "r" => {
+                let range_vec = s[1..]
+                    .split('-')
+                    .map(|x| {
+                        x.parse()
+                            .expect("Invalid bridge mode arg: range format invalid expected (x:y)")
```

Comment:
I don't understand what you mean by `expected (x:y)` ... The correct format for range is `backfill:r10-100`. So user seeing this message might be confused and think that `10:100` is a valid range selector.
If this error message is displaying, then the user probably already knows about the `backfill:r` prefix, and just messed up formatting the range selector, but in this case I think it's better to be explicit and just restate the entire format, aka `range format invalid, expected backfill:rX-Y (eg. backfill:r10-100)`